postgres-client-builder/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Declare Ubuntu and Postgres versions
-ARG UBUNTU_VERSION=20.04
```

Comment:
Just checking, but did you intend to drop the Ubuntu version? Is 20.04 not FIPS compliant?

Reply:
I did because `18.04` is the basis of our Phusion image, not `20.04`, and `20.04` messes with some of the Postgres dev packages names to redirect things to Postgres 12.

Let me take another look at that though.

Reply:
@jvanderhoof When you have a chance to take another look, I was able to keep the build image at 20.04. This has me a hair nervous about using it in the `18.04` Phusion image, but we'll deal with it if it becomes a problem.